src/parse_logcfg.c

```diff
@@ -1423,6 +1424,20 @@ int parse_logcfg(char *inputbuffer) {
 	}
 	case 148: {
 	    change_rst = 1;
```

Comment:
how about boolifying `change_rst` and moving it to `change_rst.c/h`?

Reply:
Good idea.